appengine/headless-chrome/README.md

```diff
@@ -0,0 +1,12 @@
+# Headless Chrome on Google App Engine
+
+This sample application demonstrates how to use Headless Chrome via the [Puppeteer](https://developers.google.com/web/tools/puppeteer/) module to take screenshots of webpages on [Google App Engine](https://cloud.google.com/appengine) Node.js [standard environment](https://cloud.google.com/appengine/docs/standard/nodejs).
+
+## Running locally
+
+* Install dependencies with `npm install` (Headless Chrome will take some time to download)
```

Comment:
I think we either need a period or extra line breaks here or this will look weird.

Reply:
I am not sure what you mean. This is just one line in a bullet point. I added a comma at the end.